MONAIAuto3DSeg/Resources/Models.json

```diff
@@ -81,6 +81,16 @@
                 { "url": "https://github.com/lassoan/SlicerMONAIAuto3DSeg/releases/download/Models/mediastinal-anatomy-3mm-v2.0.0.zip" }
             ]
         },
+        {
+            "title": "Prostate - Multisequence",
+            "description": "Segments peripheral (PZ) and transition zone (TZ) on T2 and ADC sequences. Trained on Medical Segmentation Decathlon dataset",
```

Comment:
Please upload sample images (one pair of corresponding T2 and and ADC images in separate nrrd files) into a new "TestData" release. I'll later add code to register them into the `Sample Data` module. This will make it easy for users to test the model.